### PR TITLE
fix: update coverage and add missing links

### PR DESCRIPTION
**Pull request by yrobla**
Signed-off-by: Yolanda Robla <yolanda@iasql.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,21 +50,30 @@ AWS is our main focus at the moment, but we plan to support GCP, Azure and other
 
 ### AWS services with significant API coverage
 
+- CloudWatch
+- CodeBuild
+- CodeDeploy
+- CodePipeline
```
Comment (dfellis) [inline README.md:56]: We have temporarily disabled this modules until @aguillenv 's Two-Way mode work is done, so I'd remove this.

```suggestion
```

File: README.md [hunk 2]
```diff
@@ -49,21 +50,27 @@ AWS is our main focus at the moment, but we plan to support GCP, Azure and other
...
-- ACM
+- API Gateway
+- CloudFront
 - Dynamo DB
-- S3
+- Elasticache
+- GraphQL
+- MemoryDB
+- [Route53](https://iasql.com/docs/aws_route53/)
+- Secrets
```
Comment (aguillenv) [inline README.md:74]:
```suggestion
- Secrets Manager
```

File: README.md [hunk 3]
```diff
@@ -49,21 +50,27 @@ AWS is our main focus at the moment, but we plan to support GCP, Azure and other
...
 ### AWS services with basic API coverage
 
-- Route53
-- ACM
+- API Gateway
+- CloudFront
 - Dynamo DB
-- S3
+- Elasticache
+- GraphQL
```
Comment (aguillenv) [inline README.md:71]: isnt this one appsync or something like that?

File: README.md [hunk 4]
```diff
@@ -49,21 +50,27 @@ AWS is our main focus at the moment, but we plan to support GCP, Azure and other
...
 
 ### AWS services with basic API coverage
 
-- Route53
-- ACM
+- API Gateway
+- CloudFront
 - Dynamo DB
-- S3
+- Elasticache
```
Comment (aguillenv) [inline README.md:70]:
```suggestion
- ElastiCache
```

**Review by depombo — APPROVED** [review]

**Review by dfellis — APPROVED** [review]

**Pull request merged**